utils/polus-imagej-util/classes/populate.py

```diff
@@ -1041,30 +1051,45 @@ def update_templates(self, ops):
         # Save a directory for the cookietin json files
         cookietin_path = cwd.joinpath('utils/polus-imagej-util/cookietin')
         
+<<<<<<< HEAD
```

Comment:
These need to be resolved.

Reply:
Merge conflicts now resolved. Code was also run to check there are no more conflicts.